opendebates/templates/base.html

```diff
@@ -142,6 +142,9 @@
   ODebates.paths.login = "{% url 'auth_login' %}";
   ODebates.paths.register = "{% url 'registration_register' %}";
 
+  ODebates.strings = ODebates.strings || {};
+  ODebates.strings.afterQuestionSubmittedText = {{ POPUP_AFTER_SUBMISSION_TEXT|safe }};
```

Comment:
Just noting that this + the `json.dumps` in context_processors seems really ugly, but was the simplest way I could think of to ensure the site doesn't break if admins enter popup text like `<span style="color: green">Thanks</span>`.